Assets/SEE/Utils/TextualDiff.cs

```diff
@@ -40,6 +41,29 @@ public static class TextualDiff
             string newRegion = FileIO.Read(newPath, newStartLine, newEndLine);
             return Diff2RichText(diff.diff_main(oldRegion, newRegion));
         }
+        /// <summary>
+        /// Returns the difference of two text regions in two version controled files.
+        /// The diff contains Rich Text markup that indicates the additions
+        /// and deletions between the two regions. An addition is text that is contained
+        /// in the new region only and a deletion is text that is contained only in the
+        /// old region.
+        ///
+        /// The Rich Text markup of a deletion renders the deleted text in red
+        /// and struck through. The markup of an additions renders the added text
+        /// in green and underlined.
+        ///
+        /// Each entry in the result is a line of text of the new region.
+        /// </summary>
+        /// <param name="old">the old region</param>
+        /// <param name="current">the new region</param>
+        /// <returns>diff in Rich Text markup</returns>
+        public static string[] DiffForDiffCity(string old, string current)
```

Comment:
This method is part of a utility class `SEE.Utils.TextualDiff`, which doesn't anything about cities in SEE. Its name should be chosen without reference to any kind of city.

Reply:
Would "VCSDiff" be a better name? Since "Diff" is already taken.

Reply:
I don't see why it has anything to do with a VCS. It just returns the diff of its parameters, wherever these strings come from.

Reply:
`Diff` is the right name. It can be overloaded. I fixed it.